pkg/madmin/README.md

```diff
@@ -262,6 +262,17 @@ Fetches Mem utilization for all cluster nodes. Returned value is in Bytes.
 | `mem.Usage.Mem`   | _uint64_ | The total number of bytes obtained from the OS         |
 | `mem.Usage.Error` | _string_ | Error (if any) encountered while accesing the CPU info |
```

Comment:
accesing -> accessing

Reply:
This is not this PR scope. Anyway fixing.